go.mod

```diff
@@ -13,18 +13,18 @@ require (
 	google.golang.org/api v0.189.0
 	gopkg.in/gcfg.v1 v1.2.3
 	gopkg.in/warnings.v0 v0.1.2
-	k8s.io/api v0.30.0
-	k8s.io/apimachinery v0.30.0
-	k8s.io/apiserver v0.30.0
-	k8s.io/client-go v0.30.0
-	k8s.io/code-generator v0.30.0
-	k8s.io/component-base v0.30.0
-	k8s.io/component-helpers v0.30.0
-	k8s.io/controller-manager v0.30.0
-	k8s.io/klog/v2 v2.120.1
-	k8s.io/kube-controller-manager v0.30.0
-	k8s.io/kubelet v0.30.0
-	k8s.io/metrics v0.30.0
+	k8s.io/api v0.31.1
+	k8s.io/apimachinery v0.31.1
+	k8s.io/apiserver v0.31.1
+	k8s.io/client-go v0.31.1
+	k8s.io/code-generator v0.31.1
+	k8s.io/component-base v0.31.1
+	k8s.io/component-helpers v0.31.1
+	k8s.io/controller-manager v0.31.1
+	k8s.io/klog/v2 v2.130.1
+	k8s.io/kube-controller-manager v0.31.1
+	k8s.io/kubelet v0.31.1
+	k8s.io/metrics v0.31.1
 	k8s.io/utils v0.0.0-20230726121419-3b25d923346b
```

Comment:
The upstream PR also updates this package, should we?

Reply:
Done.